docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -10,38 +21,122 @@ name servers must have addresses in the routable public addressing space.
 
 IANA is responsible for global coordination of the IP addressing system.
 Aside its address allocation activities, it maintains reserved address ranges
-for special uses. These ranges can be categorized into three types : 
-[Special purpose IPv4
-addresses](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml),
-[Special purpose IPv6
-addresses](https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xml)
-and [Multicast reserved
-addresses](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xml).
+for special uses. These ranges can be categorized into three types:
+
+* [Special purpose IPv4 addresses]
+* [Special purpose IPv6 addresses]
+
+### Scope
+
+IP addresses from authoritative name servers are matched against IANAs list of 
+public routable address ranges.
 
 ### Inputs
 
-The domain name to be tested.
+* The domain name to be tested.
+
+### Summary
+
+Message Tag                       | Level    | Arguments | Message ID for message tag
+:-------------------------------- |:---------|:----------|:--------------------------
+A01_ADDR_GLOBALLY_REACHABLE       | INFO     |           | IP addresses listed as globally reachable.
+A01_NO_GLOBALLY_REACHABLE_ADDR    | CRITICAL |           | None of the IP addresses listed as globally reachable.
+A01_ADDR_NOT_GLOBALLY_REACHABLE   | ERROR    | ns_list   | IP adress for "{ns}" not listed as globally reachable.
+A01_DOCUMENTATION_ADDR            | ERROR    | ns_list   | IP adress for "{ns}" part of address range for documentation purposes.
+A01_LOCAL_USE_ADDR                | ERROR    | ns_list   | IP adress for "{ns}" part of address range intended for local use on network or service provider level..
```

Comment:
The message ID should be updated with "{ns_list}". 

I also suggest that the message is reworded so that "ns_list" can be put last i the message ID. The list of name servers will break the reading of the textual message. For how "ns_list" is put at the end of message ID see e.g. [messages in specification of Zone11]. (Single name server in "ns" can be put in the middle of the message ID.)

[messages in specification of Zone11]: https://github.com/zonemaster/zonemaster/blob/master/docs/public/specifications/tests/Zone-TP/zone11.md#summary